src/SourceCode.Clay.OpenApi/Components.cs

```diff
@@ -0,0 +1,162 @@
+using SourceCode.Clay.Collections.Generic;
+using System;
+using System.Collections.Generic;
+
+namespace SourceCode.Clay.OpenApi
+{
+    /// <summary>
+    /// Holds a set of reusable objects for different aspects of the OAS. All objects
+    /// defined within the components object will have no effect on the API unless they
+    /// are explicitly referenced from properties outside the components object.
+    /// </summary>
+    public class Components : IEquatable<Components>
```

Comment:
sealed?

Reply:
Nope, nothing in the lib can be sealed - inheritance is how vendor extensions can be added.